src/webgpu/api/validation/createTexture.spec.ts

```diff
@@ -530,7 +530,9 @@ g.test('texture_size,2d_texture,compressed_format')
   });
 
 g.test('texture_size,3d_texture,uncompressed_format')
```

Comment:
The test name still also needs to be updated. It says "texture_size" but it is testing more than just that.

Reply:
Good point. The name is not accurate if so. Considering tests nearby are all testing texture_size, it may think it makes more sense to keep the name but remove depth/stencil formats for 3D textures because they have already been tested in another test. 

I keep the name as uncompressed_formats (not regular_format) to show we don't miss any formats (uncompressed formats + compressed formats below = all formats), and add some statement to explain that we are testing regular formats because depth/stencil formats are invalid for 3D textures.

Reply:
> because they have already been tested in another test

Where are these tests? I thought the reason you were making this PR was because we didn't have any yet.

Reply:
As I mentioned above, another test in this test file named dimension_type_and_format_compability can cover all valid/invalid formats for 3D textures (and other texture dimension types).

Reply:
Ok, these are new tests then? If so, we should either either add them, or add a TODO/.unimplemented(), and should do so in this PR so the work is tracked now.

Reply:
No new tests. dimension_type_and_format_compatibility() is already there in this file, which have already covered formats and dimension types (1d, 2d, and 3d) compatibility. As a result, depth/stencil formats for 3D textures (which are invalid) in this test are totally redundant, so I think we can remove them.

Reply:
> dimension_type_and_format_compatibility() is already there in this file

Ohh, sorry. I copied and pasted "dimension_type_and_format_compability" into the search box but didn't notice it has a very subtle typo in it!

Yes, I understand now, all looks good. Sorry for the delay!

Reply:
Sorry for the typo... Thanks for your review!

Reply:
Can you roll CTS into Chromium? A Chromium patch is pending on this PR. Alternatively, I can roll WebGL CTS into Chromium.

Reply:
It rolls automatically:
https://autoroll.skia.org/r/webgpu-cts-chromium-autoroll
however the latest roll is failing, apparently due to the commit right before yours...

Reply:
Got it. No worries. I can keep an eye on when this PR is rolled.